p2p_test.go

```diff
@@ -86,7 +88,7 @@ func TestUnicast(t *testing.T) {
 
 	for i, host := range hosts {
 		neighbors := host.Neighbors(ctx)
-		require.True(t, len(neighbors) > 0)
+		require.True(t, len(neighbors) >= n/3)
```

Comment:
after connecting into p2p network, should at least connect to a portion of peers
test shows n/2 fails sporadically, n/3 is consistent success